linetools/spectra/io.py

```diff
@@ -25,7 +25,8 @@
 
 
 def readspec(specfil, inflg=None, efil=None, verbose=False, multi_ivar=False,
-             format='ascii', exten=None, debug=False, select=0, **kwargs):
+             format='ascii', exten=None, debug=False, select=0,
+             head_ext=0, **kwargs):
```

Comment:
Could you please move the `head_ext` keyword right after `exten` keyword. Consider refactor `exten` -> `ext` (?)